### ISSUE TITLE
Adapt BISR to Physical-Cognitive track

### ISSUE THREAD

**Issue by andkov** [IALSA/IALSA-2015-Portland#104]
by @ampiccinin : 
> Since most of the comparisons we are making are within domains across datasets, it would be nice if the BISR graphs could also be arranged that way .

![img_5644](https://cloud.githubusercontent.com/assets/2608073/10984090/0b4d4328-83cc-11e5-9e03-7e89acd43a62.JPG)

**Comment by wibeasley** [comment]
For some reason, the picture is upside-down in the issue, but right-side up at https://cloud.githubusercontent.com/assets/2608073/10984090/0b4d4328-83cc-11e5-9e03-7e89acd43a62.JPG

**Comment by andkov** [comment]
yes, i've noticed, doesn't go away after reupload. i'll let it be for now, but curious what's up.

**Comment by wibeasley** [comment]
I'm wondering if there's some metadata encoded from the phone (about which direction is up) that one of them correctly interprets, but the other doesn't.  I'm completely guessing though.

**Comment by ampiccinin** [comment]
…yeah…held the phone at a weird angle to get the shot. Sorry – did not mean to cause problems. Was running off and did what it took to send the email before I left. A

**Comment by andkov** [comment]
@wibeasley , you were right: copy/pasted the imaged into a new png and it's ok.

**Comment by andkov** [comment]
@ampiccinin , what do numbers in your sketch stand for? point estimates?

**Comment by ampiccinin** [comment]
☺ those are the correlations from the BISR graph (yes, point estimates).  They are the re-done ones, right?

Underline is p < .10

Box is p < .05

Underlined box is p < .01

Double box is < .001

I’ve emailed Debbie about the “NA” and requested output where it says “requested”.

A

**Comment by andkov** [comment]
@ampiccinin , no, they are not re-done. They are computed by an R script from the estimates of the covariances before the Wellcome conference. At that time we didn't use a direct estimates of correlation in MPlus. We wanted to see if R scripts produce the same estimates as Mplus scripts, but we never really had the time to check is systematically and just switched to estimating it with Mplus to be consistent.  Now that we've seen that Mplus produces weird correlation estimates too I'm more confident to say they R and Mplus estimates are equivalent, however, I have not yet produced a demonstration of this.

**Comment by ampiccinin** [comment]
Do I recall that there was a v.1 and v.2 of the R-computed correlations and CIs, though?

If not, then could you please coordinate with @annierobi and @casslbrown to test a subset so we will know whether we can be comfortable with the estimates currently in the BISR graph?

I would say that – apart from the models they are running now – this is the most urgent task.

I can manage the rest by hand, as I did the PPP.

Thanks!
A

**Comment by ialsa-lab** [comment]
I will have to do it myself, so if the rest can wait, I'll get on this one.

Sent from my iPhone, please excuse typing and T9 errors.

On Nov 5, 2015, at 4:33 PM, Andrea Piccinin <notifications@github.com<mailto:notifications@github.com>> wrote:

Do I recall that there was a v.1 and v.2 of the R-computed correlations and CIs, though?

If not, then could you please coordinate with @annierobi and @casslbrown to test a subset so we will know whether we can be comfortable with the estimates currently in the BISR graph?

I would say that - apart from the models they are running now - this is the most urgent task.

I can manage the rest by hand, as I did the PPP.

Thanks!
A

**Comment by andkov** [comment]
@ampiccinin , to clarify your graph specification. You would like to have all cognitive measures to be display next to each other? With each measure given six columns, we are talking about 40+ columns. . So just making sure that a table of these dimensions ( rows = number of studies; columns = number of  by cognitive measures * 6) is what you have in mind.

**Comment by ampiccinin** [comment]
No worries for now. Let’s talk about it when we meet tomorrow.

Thanks! Best, Andrea

**Comment by ampiccinin** [comment]
To provide a record of our discussion, we would just want to display individual variables, generally, or at most individual domains, but across all studies.
